test/index.js

```diff
@@ -1,25 +1,29 @@
 /* eslint-env mocha */
+/* eslint-disable prefer-arrow-callback  */
+/* eslint-disable func-names  */
```

Comment:
Could you just name the anonym fn with "test" instead? You should be able to remove these 2 eslint rules after that